pyproject.toml

```diff
@@ -1,12 +1,19 @@
 [build-system]
-requires = ["setuptools", "wheel", "setuptools-rust"]
+requires = [
+    "setuptools>=77.0",
```

Comment:
Should we specify this in `requirements-dev.txt` too, which requires `setuptools`?